### DISCUSSION TITLE
No build parallelism

### DISCUSSION THREAD

**Discussion by RobertHenry6bev** [duckdb/duckdb#6708] [General]
I can not figure out how to compile duckdb in parallel.  The majority of the compiles are done sequentially, regardless of the value of the -j $N flag given to the manually invoked top-level make.  Some sub-projects are build in parallel within the sub-project.

it takes about 3 hours to grind through the whole build, which makes it difficult to experiment with compilers.

**Comment by Mause** [comment]
We use the ninja-build tool rather than make for our builds to ensure build parallelism

You should be able to run it like this: `GEN=ninja make debug`
